Index: sirius-0.8.0/sirius.desktop
===================================================================
--- sirius-0.8.0/sirius.desktop
+++ sirius-0.8.0/sirius.desktop	2009-05-13 18:40:05.000000000 +0200
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Sirius
 Name[de]=Sirius
 Name[fr]=Sirius
@@ -10,4 +9,4 @@
 Icon=sirius.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;
+Categories=Game;BoardGame;
